### ISSUE TITLE
error logging how to

### ISSUE THREAD

**Issue by bskade** [pforret/crontask#2]
hi,
nice tool - serves us very well
but as far I can see, errors will not be promted
is there any way to activate this

my tmp solution so far is just to add 
 cat "$f_stderr"
in the error case

**Comment by pforret** [comment]
In case of an error, the 'failure' URL is called (if defined)
Would you like the option that in case of an error, the script fails too?

Example:

`./crontask -y http://www.example.com/success -n "https://www.example.com/failure" -v url "https://www.google.nocom"`

```
(...)
# Executing:    ['/usr/bin/curl' -s 'https://www.google.nocom'] 
# Using shell:  [/opt/homebrew/bin/bash] 
# Call to [https://www.example.com/failure]: OK 
20
0
«https://www.google.nocom»: msecs lines [error]
ERROR: real 0.02
# crontask finished after 1 seconds
```

**Comment by bskade** [comment]
I just searched for an option, so that the error output of the failing script will be logged/printed as well

**Comment by pforret** [comment]
I've released a patch: v3.0.3

Upon success, nothing changes:

```
$ crontask cmd "sleep 1"
1010
0
«sleep 1»: msecs lines
2021-12-21 21:56:37
```

Upon failure, the error message is shown on the 4th line

```
$ crontask cmd "sleeq 1"
0
0
«sleeq 1»: msecs lines [error]
ERROR: /opt/homebrew/bin/bash: line 1: sleeq: command not found
```

**Comment by bskade** [comment]
👌perfect  - thank you
